<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewHighscore</class>
 <widget class="QDialog" name="DialogNewHighscore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Highscore</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_OK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_name"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Enter your name:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QTableView" name="table_highscore">
       <property name="minimumSize">
        <size>
         <width>283</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>283</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_congrats">
       <property name="text">
        <string>Congratulations you got a new highscore!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
